packages/storybook/src/themes/V7/AzureDarkTheme.ts

```diff
@@ -0,0 +1,7 @@
+import { AzureCustomizationsDark } from '@uifabric/azure-themes';
```

Comment:
Probably the v7 folder name should be lowercase to more closely match the convention elsewhere.